### DISCUSSION TITLE
Instructions about installing karmada in AWS EKS re using the eks api server and etcd

### DISCUSSION THREAD

**Discussion by d4rkhunt33r** [karmada-io/karmada#4518] [Q&A]
Hi everyone

I am looking for guidance about how to install karmada in aws eks without deploying my own karmada api server and etcd

I know is posible because in the frequently asked questions it says some user are doing it but i can't find find documentation about it

using eks control plane will facilitate the administration of the api server and etcd

thank you so much for your attention

**Comment by likku123** [comment]
By now you might have figured it out but sharing for others who visits this page :)

https://aws-solutions-library-samples.github.io/compute/multi-cluster-application-management-with-karmada-and-amazon-eks.html

https://aws.amazon.com/solutions/guidance/multi-cluster-application-management-with-karmada-and-amazon-eks/

**Reply by RainbowMango** [reply]
Thanks @likku123. This is amazing!
